src/target/riscv/riscv.c

```diff
@@ -732,16 +732,20 @@ static struct match_triggers_tdata1_fields fill_match_triggers_tdata1_fields_t2(
 static struct match_triggers_tdata1_fields fill_match_triggers_tdata1_fields_t6(
 		struct target *target, struct trigger *trigger)
 {
-	RISCV_INFO(r);
+	bool misa_s = riscv_supports_extension(target, 'S');
+	bool misa_u = riscv_supports_extension(target, 'U');
+	bool misa_v = riscv_supports_extension(target, 'V');
```

Comment:
Is this intended to be checking for the hypervisor extension?

Reply:
Right. The virtualization extension.

Good catch. Thanks.